shopinvader_invoice/services/invoice.py

```diff
@@ -68,39 +68,39 @@ def _get_return_invoice_schema(self):
         :return: dict
         """
         invoice_schema = {
-            "invoice_id": {"type": "integer"},
-            "number": {"type": "string"},
-            "date_invoice": {"type": "string"},
+            "move_id": {"type": "integer"},
```

Comment:
IMO: To keep compatibility between version and as the link between Odoo and Json values should stay abstract, I prefer to let this schema as origin and only edit the parser to match with this structure.